content/training/Walk-through/place_analyses.Rmd

```diff
@@ -371,18 +393,38 @@ cases <- cases[(cases$epiweek == "2016-W26" |
                cases$epiweek == "2016-W27" | 
                cases$epiweek == "2016-W28" |
                cases$epiweek == "2016-W29" |
-               cases$epiweek == "2016-W30"), ]
+               cases$epiweek == "2016-W30" | 
+               cases$epiweek == "2016-W31" |
+               cases$epiweek == "2016-W32" |
+               cases$epiweek == "2016-W33" |
+               cases$epiweek == "2016-W34" |
+               cases$epiweek == "2016-W35" |
+               cases$epiweek == "2016-W36" |
+               cases$epiweek == "2016-W37" |
+               cases$epiweek == "2016-W38" |
+               cases$epiweek == "2016-W39" |
+               cases$epiweek == "2016-W40" |
+               cases$epiweek == "2016-W41" |
+               cases$epiweek == "2016-W42" |
+               cases$epiweek == "2016-W43" |
+               cases$epiweek == "2016-W44" |
+               cases$epiweek == "2016-W45" ), ]
```

Comment:
I know this is not being shown to the users, but just a note on style here. 

ISO 8601 dates (including weeks) will always automatically sort themselves, so you can simply do a logical comparison:

```r
cases <- cases %>%
  dplyr::mutate(epiweek = as.character(epiweek)) %>%
  dplyr::filter(epiweek >= "2016-W26") %>%
  dplyr::filter(epiweek <= "2016-W45") %>%
  dplyr::mutate(epiweek = forcats::fct_inorder(epiweek))
```

Reply:
I tried to make this change but get this error:
Error: Column `epiweek` can't be modified because it's a grouping variable

do you know what this is about?

Reply:
@zkamvar  I tried to make this change but get this error:
Error: Column `epiweek` can't be modified because it's a grouping variable

do you know what this is about?

Reply:
Got it!  Added ungroup() %>%